src/api/v1/bag/nummeraanduiding/getNummeraanduidingByAddress.ts

```diff
@@ -0,0 +1,38 @@
+import joinUrl from '../../../../app/utils/joinUrl'
+import environment from '../../../../environment'
+import queryStringParser from '../../../../shared/services/query-string-parser/query-string-parser'
+import { fetchWithoutToken } from '../../../../shared/services/api/api'
+import { Root as Nummeraanduiding } from './types'
+
+type ReceiveFields = {
+  _fields: string
+}
+
+/**
+ * Retrieve nummeraanduiding
+ *
+ * API documentation: https://api.data.amsterdam.nl/v1/bag/nummeraanduiding
+ *
+ * @param {String} queryParams - Full URL or search string
+ * @param {String} receiveFields - comma separated (no spaces) values indicating which fields should be returned from the API request
+ */
+// eslint-disable-next-line import/prefer-default-export
+export const getNummeraanduidingByAddress = (
+  queryParams: string,
+  receiveFields?: string,
+): Promise<Nummeraanduiding> | null => {
+  const paramsString = queryParams.substr(queryParams.indexOf('?'))
+  const searchParams = new URLSearchParams(queryStringParser(paramsString))
+
+  if (!searchParams.get('postcode') || !searchParams.get('huisnummer')) {
+    return null
+  }
+
+  if (receiveFields) {
+    searchParams.append('_fields', receiveFields)
+  }
+
+  const url = joinUrl([environment.API_ROOT, 'v1/bag/nummeraanduiding'])
+
+  return fetchWithoutToken(`${url}?${searchParams.toString()}`)
```

Comment:
If you do not want to do string concatenation I believe you could also use the `URL` constructor here. 
```suggestion
  const url = new URL(joinUrl([environment.API_ROOT, 'v1/bag/nummeraanduiding']))
  
  url.search  = searchParams.toString()

  return fetchWithoutToken(url.toString())
```

Reply:
I don't see a benefit with the proposed change.

Reply:
The benefit is that you are leaving *all* the parsing and formatting logic for URLs to the built-in classes that are provided by the browser.

Reply:
In your example, search param parsing happens twice, because of `searchParams.toString()` and setting the result of that to `url.search`. I'm not inclined to apply the proposed change here.

Reply:
It's not that important, either way is fine. Parsing should not occur twice per-se, that all depends on the internal API of `URL`. Besides that, it's not like we're calling this method on any budget so it doesn't really matter if it was parsed multiple times.